### DISCUSSION TITLE
SPA mode how to use babel plugin?

### DISCUSSION THREAD

**Discussion by afx919** [quasarframework/quasar#15231] [CLI - PWA mode] [answered]
I want to use the babel plugin like 'babel-plugin-transform-remove-console', but it doesn't work, console output is not deleted.

babel.config.js:
```
module.exports = (api) => {
  return {
    presets: [
      [
        "@quasar/babel-preset-app",
        api.caller((caller) => caller && caller.target === "node")
          ? { targets: { node: "current" } }
          : {},
      ],
    ],
    plugins: ["transform-remove-console"],
  };
};
```
How do I configure the babel plugin to take effect? It works in the Quasar V1 version, but not work in the  Quasar V2.

**Comment by yusufkandemir** [comment, marked as answer]
I don't know about the babel plugin, but this should work:
```js
// quasar.config.js > build
uglifyOptions: {
  compress: {
    drop_console: true
  }
}
```
If you need the babel plugin for whatever reason, create a reproduction.

**Reply by afx919** [reply]
I just want to delete the console output, It works! Thank you!
